src/screens/IndexScreen.js

```diff
@@ -390,6 +397,21 @@ export const IndexScreen = ({ navigation, route }) => {
                 <ListComponent
                   ListHeaderComponent={
                     <>
+                      {query === QUERY_TYPES.EVENT_RECORDS && !!eventListIntro && !showCalendar && (
+                        <>
+                          <Wrapper>
+                            <RegularText small>{eventListIntro.introText}</RegularText>
+                          </Wrapper>
+                          {eventListIntro.url && eventListIntro.buttonTitle && (
```

Comment:
add !! for the checks here please to be safe here for all platforms

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/c5eb0ce08630b8f7884465ae161431309b8946ef